docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+services:
+  app-server:
+    image: 'yashuyellapu/employee-portal-fullstack:backend-latest'
```

Comment:
when the image tag is the same, docker or ansible will not pull the latest image - assuming that the image is the same as the tag is the same. How can we solve this?